src/main/scala/org/broadinstitute/dsde/agora/server/webservice/AgoraService.scala

```diff
@@ -64,9 +64,7 @@ abstract class AgoraService(permissionsDataSource: PermissionsDataSource) extend
               val deletionAttempt = agoraBusiness.delete(targetEntity, entityTypes, username)
 
               onComplete(deletionAttempt) {
-                case Success(rowsDeleted) => complete("FIX ME")
-                // TODO[GAWB-3052] The line above should be the line below; figure out what implicit may be missing
-                // case Success(rowsDeleted) => complete(rowsDeleted)
+                case Success(rowsDeleted) => complete(rowsDeleted.toString)
```

Comment:
There is no marshaller for `Int`, but the original code was calling `toString` on the rows value.

Reply:
I'm pretty sure there's no `RootJsonFormat` for `Int`, even though there's a `JsonFormat` for it (and the *Formats get turned into *Marshallers eventually). That's because a raw int isn't a valid json packet - it needs to be inside an object or an array.  So, for instance, `complete(Seq(rowsDeleted))` would work - but that's not what we want to return.

i.e. the business logic is wrong here, not the marshallers. I'm fine with using `toString` as a workaround.

Reply:
If the business logic is wrong here, then it was wrong in the original code as well. I suppose we could story this topic for an improvement at some point if we're all OK with this approach.